classes/Kohana/Jam/Range.php

```diff
@@ -34,8 +34,13 @@ public static function sum(array $ranges, $format = NULL)
 
 		foreach ($ranges as $range)
 		{
-			$min += $range->min();
-			$max += $range->max();
+			if (!empty($range->min())) {
```

Comment:
I thought initially it could be done this way. Just wasn't sure it would be clear enough. However this way is much more simpler.